THIRDPARTY_LICENSE.txt

```diff
@@ -113,3 +113,16 @@ Copyright 2005 Joshua Tauberer <http://razor.occams.info>
 
 Feel free to use this file however you want, but
 credit would be nice.
+
+*******************************************************************************
+*******************************************************************************
+tweetnacl-js `nacl-fast.js` `nacl-util.js`
```

Comment:
I think it would be good to also adept the readme, as the other included libraries are listed there too.

Reply:
Done

Reply:
I was more thinking about adding the new libs to the readme but completely removing that part is ok for me too.